pages/1.9/release-notes/1.9.7/index.md

```diff
@@ -0,0 +1,215 @@
+---
+layout: layout.pug
+navigationTitle:  Release Notes for 1.9.7
+title: Release Notes for 1.9.7
+menuWeight: 0
+excerpt:
+---
+
+These are the release notes for DC/OS 1.9.7.
+
+[button color="purple" href="https://downloads.dcos.io/dcos/stable/1.9.7/dcos_generate_config.sh"]Download DC/OS Open Source[/button]
+
+# Issues Fixed in DC/OS 1.9.7
+
+- CORE-1375 - Docker executor no longer hangs due to lost messages.
+- DCOS_OSS-1943 - Modified the LIBPROCESS_WORKER_THREADS environment variable to 16 to increase the minimum number of worker threads.
+
+# Notable Changes in DC/OS 1.9.7
+
+- Mesos 1.2.3 is integrated with DC/OS 1.9.7.
+- Marathon 1.4.11 is integrated with DC/OS 1.9.7.
+
+# Issues Fixed in DC/OS 1.9.6
+
+- COPS-980 - Marathon's default task unreachable behavior reinstated.
+- DCOS-17947 - [`cluster_docker_credentials`](/1.9/installing/oss/custom/configuration/examples/#dcos-cluster-with-three-masters-an-exhibitorzookeeper-managed-internally-custom-docker-credentials-two-private-agents-and-google-dns) example corrected.
+- DCOS-19217 - Metronome adheres to crontab standard.
+- DCOS-19453 - TLS compression disabled because TLS is vulnerable to the CRIME attack. [enterprise type="inline" size="small" /]
+- DCOS-19508 - Non-superusers can now edit an app that uses secrets. [enterprise type="inline" size="small" /]
+- DOCS-2130 - [Security hardening guide](/1.9/security/ent/hardening/) improvements. [enterprise type="inline" size="small" /]
+- DOCS-2153	- [Metrics documentation](/1.9/metrics/) improvements.
```

Comment:
nit: too many spaces after the ticket number.

Reply:
Formatting is consistent with previous versions of release notes.